src/i18n/en/index.ts

```diff
@@ -1,6 +1,7 @@
 import type { BaseTranslation } from '../i18n-types';
 
 const en = {
+	allModels: 'All models',
```

Comment:
Let's change this to `All other models` so the description matches the new behavior.

Reply:
Okay, so the recently used models shouldn't be included in the second group of models?

Reply:
Correct